packages/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -69,3 +86,8 @@ exports.inputUserNote = userNote => {
 exports.submit = () => {
     goToPaymentButton().click();
 };
+
+exports.getCharacterLength = (userNote) => {
```

Comment:
Should we rename this function to `getUserNoteLength`?

Alternatively, put this in a helper file where we can make this function more generic/reusable by passing in any selector and import it into this component object. Thoughts @Festivejelly?

Reply:
Can we add JSDoc description?

Reply:
yep will do @siggerzz. Which functions require JSDoc descriptions? Is it any function that's more complex than the normal .isDisplayed() or .getText()?

Reply:
I really like the idea of having a helper file with generic functions - would be so useful for future tests 👍